index.html

```diff
@@ -144,7 +172,8 @@ <h1>
                 <span>Skills</span>
                 <span>&#10095;</span>
             </div>
-        </section>
+        </div>
+    </div>
     </footer>
     <section class="form_section">
```

Comment:
- [ ] You're yet to implement the request of the previous about the background image you used. It's kind of not tally with the Figma template. Kindly look into it and fix.

### Your Design
![your-form-area](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/26597432/fffcd168-37ef-4411-a746-b8dcc16d2432)

### Figma Design
![figma-form-area](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/26597432/3c7f8aa5-a566-45c3-8668-f2cef93aea00)

Reply:
Resolved thanks for you support